src/view_page.cpp

```diff
@@ -1208,6 +1208,10 @@ void View::DrawSystemEditorialElement(DeviceContext *dc, EditorialElement *eleme
     // if (element->m_visibility == Visible) {
     //    DrawSystemChildren(dc, element, system);
     //}
+    if (element->Is(ANNOT)) {
+        Annot *annot = dynamic_cast<Annot *>(element);
```

Comment:
I think we want to have a DrawAnnot method to be called from all DrawXXXEditorialElement. Eventually dealing with visible / invisible annot will be done in that method